src/couch_replicator_httpc.erl

```diff
@@ -406,3 +422,33 @@ after_redirect(RedirectUrl, _Code, HttpDb, Params) ->
 after_redirect(RedirectUrl, HttpDb, Params) ->
     Params2 = lists:keydelete(path, 1, lists:keydelete(qs, 1, Params)),
     {HttpDb#httpdb{url = RedirectUrl}, Params2}.
+
+
+backoff_key(HttpDb, Params) ->
+    Method = get_value(method, Params, get),
+    Url = HttpDb#httpdb.url,
+    {Url, Method}.
```

Comment:
Any chance for a replication job we could have duplicate entry? Off the top of my head I can't think of any since each rep should be it's own process + ets table.